tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -197,17 +215,21 @@ private:
    /// Throws an RException if fDefaultEntry is nullptr
    void EnsureNotBare() const;
 
-   RNTupleModel();
+   /// The field name can be a top-level field or a nested field. Returns nullptr if the field is not in the model.
+   Detail::RFieldBase *FindField(std::string_view fieldName) const;
```

Comment:
What is the difference between `FindField` and `GetField`?

Reply:
The only difference is that `GetFields` returns a const ref and throws if the field does not exist, `FindField` is the internal interface with a pointer (null pointer if the field is not there). Hm, we could name them both `GetField`...